### PR TITLE
Feature/logic

### PR DESCRIPTION
**Pull request by YoseptF**
# Restaurant Page
front

This is an example of how to make a page almost completly completly out of js.

---

## Local installation

1. Clone this repo
```
$ git clone git@github.com:YoseptF/To-do-List.git || $ git clone https://github.com/YoseptF/To-do-List.git
``` 
2. cd into the correct branch
```
$ cd [your desired branch]
``` 
2. install the dependencies
```
$ npm install
```
3. start the project
```
$ npm start
```

4. Open your localhost
It will automatically open a browser tab, but if for some reason it doesn't, you can go to
```
http://localhost:3000/
```

### CODE REVIEW

File: src/packages/project.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  checkBtn.innerHTML = 'check';
+  addBtn.innerHTML = 'add';
+
+  proj.append(buttonWrapper);
+  buttonWrapper.append(deleteBtn);
+  buttonWrapper.append(addBtn);
+  buttonWrapper.append(checkBtn);
+
+  DOMListener('.add-btn', 'click', displayTodoForm);
+};
```
Comment (maelfosso) [inline src/packages/project.js]: Must not be here!
You should separate the DOM with Logic.

File: src/packages/project.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const date = DOMcreate('h3', 'date');
+    date.innerHTML = id;
+
+    item.append(h2);
+    item.append(date);
+
+    DOMappend('.board', item);
+
+    buttons(item);
+  };
```
Comment (maelfosso) [inline src/packages/project.js]: Must not be here!
You should separate the DOM with Logic.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @YoseptF @YairFernando67 
Wonderful UI/UX experienced :smile: It's cool
However, I got some errors.

1. From the `Default Project`, I add a TODO and when clicking on **Check** button, I got this error
Screenshot from 2020-04-17 01-45-57
Because of this error, I can't check the other features.

2. We can delete a project but The Default project should always be available.

3. Please, re-read the SOLID principles [here](http://aspiringcraftsman.com/2011/12/08/solid-javascript-single-responsibility-principle/), because you are mixing DOM with Logic.

Kindly make the suggested changes and submit a new code review request.
Thanks

Mael FOSSO

**Review by azdrenymeri — APPROVED** [review]
### Hello @YoseptF :wave: 
#### All required changes are implemented :heavy_check_mark: 
#### You can merget it.

**Pull request merged**